### PR TITLE
Update Tug-of-War Scripts

### PR DESCRIPTION
**Pull request by jimmyDunne**
Fixes issue #2372

### Brief summary of changes
- Changed order of front matter so Help is useful (doesn't just show the license)
- Added inertia to the model
- Made scripts look pretty 

### Testing I've completed
- Ran scripts locally and worked fine. The visualizer doesn't display the model geometry, but that is a separate issue

### Looking for feedback on...
- How does my script look?

### CHANGELOG.md (choose one)
- no need to update because it fixes 🐛

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2376)

### CODE REVIEW

File: Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m [hunk 1]
```diff
@@ -1,58 +1,49 @@
...
 % The OpenSim API is a toolkit for musculoskeletal modeling and           %
 % simulation. See http://opensim.stanford.edu and the NOTICE file         %
 % for more information. OpenSim is developed at Stanford University       %
 % and supported by the US National Institutes of Health (U54 GM072970,    %
 % R24 HD065690) and by DARPA through the Warrior Web program.             %
-%                                                                         %   
-% Copyright (c) 2005-2017 Stanford University and the Authors             %
-%                                                                         %   
+%                                                                         %
+% Copyright (c) 2005-2019 Stanford University and the Authors             %
```
Comment (chrisdembia) [inline Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m]: Forward–thinking.

File: Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m [hunk 2]
```diff
@@ -1,79 +1,64 @@
...
+% Instantiate a Free Joint (6 DoF) that connects the block and ground.
 blockSideLength      = 0.1;
-locationInParentVec3 = ArrayDouble.createVec3([0, blockSideLength/2, 0]);
+locationInParentVec3 = Vec3(0, blockSideLength/2, 0);
 blockToGround        = FreeJoint('blockToGround', ...
                             ground, locationInParentVec3, zeroVec3, ...
                             block, zeroVec3, zeroVec3);
 
-% Set bounds on coordinates
+% Set bounds on the 6 coordinates of the Free Joint.
```
Comment (tkuchida) [inline Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m]: Briefer (though perhaps less clear):
<pre>
for i=0:2, blockToGround.upd_coordinates(i).setRange(angleRange); end
for i=3:5, blockToGround.upd_coordinates(i).setRange(positionRange); end
</pre>

File: Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m [hunk 3]
```diff
@@ -86,73 +71,73 @@
...
 muscleController.setName('LinearRamp_Controller')
 muscleController.setActuators(model.updActuators())
 
-% Define linear functions for the control values for the two muscles
+% Define linear functions for the control values
 slopeAndIntercept1=ArrayDouble(0.0, 2);
 slopeAndIntercept2=ArrayDouble(0.0, 2);
 
-% Muscle1 control has slope of -1 starting 1 at t = 0
+% Set Muscle1 to have a control slope of -1 starting 1 at t = 0
```
Comment (tkuchida) [inline Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m]: It looks like the slope will be `-1/3`, not `-1` as indicated by the comment.

File: Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m [hunk 4]
```diff
@@ -86,73 +71,73 @@
...
 slopeAndIntercept1.setitem(0, -1.0/(finalTime-initialTime));
 slopeAndIntercept1.setitem(1,  1.0);
 
-% Muscle2 control has slope of 0.95 starting 0.05 at t = 0
+% Set Muscle2 to have a control slope of 0.95 starting 0.05 at t = 0
```
Comment (tkuchida) [inline Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m]: Slope of `0.95/3`?

File: Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m [hunk 5]
```diff
@@ -1,58 +1,49 @@
...
+tool = ForwardTool();
+tool.setName('tugOfWar');
+% Set the model on that the forward tool will operate on.
 tool.setModel(osimModel);
-
-% Define the start and finish times for simulation
+% Set the start and end times of the simulaiton
 tool.setStartTime(0);
 tool.setFinalTime(3);
+% Set the simulation to solve for euilibrium states of the muscle.
```
Comment (tkuchida) [inline Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m]: euilibrium -> equilibrium

File: Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m (hunk 4 above)
Comment (jimmyDunne) [reply Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m]: Yeah, weird given the comment. I have tried to fix the coefficients to match the comment, and didn't get a compelling simulation— the block essentially stays in place. So instead, I will push changes that just 'fix' (delete) the comment.

File: Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m (hunk 2 above)
Comment (jimmyDunne) [reply Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m]: I like this suggested way of doing it, so included it.

File: Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m (hunk 1 above)
Comment (jimmyDunne) [reply Bindings/Java/Matlab/examples/TugOfWar_CompleteRunVisualize.m]: I haven't thought enough to licenses, but I don't understand why you can't just put something like '2005-2300' haha

**Comment by clnsmith** [comment]
@jimmyDunne 
Just tested the OpenSimCreateTugOfWarModel.m script in the OpenSim 4.0 download available from the opensim website. 

It looks like your changes are not included in the distributed files (which makes sense since the date on the distribution is Nov 13, 2018, and the merge is Nov 19, 2018).

But I think you are still missing the call to add the joint to the model:
model.addJoint(blockToGround)

OpenSim automatically adds a FreeJoint anyways (see output below), so its not a big issue but good to fix as an example.

Body 'Block' not connected by a Joint.
A FreeJoint will be added to connect it to ground.

**Pull request merged**